source/client-side-encryption/client-side-encryption.rst

```diff
@@ -1079,7 +1079,8 @@ update                   AUTOENCRYPT
 authenticate             BYPASS
 getnonce                 BYPASS
 logout                   BYPASS
-isMaster                 BYPASS
+hello                    BYPASS
```

Comment:
This will require a change in libmongocrypt as well so the "hello" command does not undergo the automatic encryption process needlessly. I've created MONGOCRYPT-316 to track that.

Reply:
Good catch!